ui/src/utils/forms.ts

```diff
@@ -1,7 +1,11 @@
 import dayjs from "dayjs";
 
-export function formatDate(date: dayjs.Dayjs | Date, showTime?: boolean) {
-  const format = showTime ? "YYYY-MM-DD HH:mm" : "YYYY-MM-DD";
+export function formatDate(
+  date: dayjs.Dayjs | Date,
+  format: "date" | "date-time" | "time" = "date"
```

Comment:
[Possibly](https://github.com/0xPolygonID/sh-id-platform/pull/334/files#r1176223171) should be `format: "date" | "date-time" | "time" = "date-time"`

Reply:
Okay, so we definitely want to display the time in almost all cases.

Reply:
Done!